orcestra_book/plans/HALO-20240810a.md

```diff
@@ -0,0 +1,69 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+platform: HALO
+flight_id: HALO-20240810a
+takeoff: 2024-01-01 00:00
+landing: 2020-01-01 01:00
+crew:
+  - name: John Doe
+    job: PI
+  - name: Emily Taylor
+    job: WALES
+  - name: Liam Brooks
+    job: HAMP
+  - name: Ava Reyes
+    job: Dropsondes
+  - name: Ethan Wallace
+    job: Smart/VELOX
+  - name: Ruby Patel
+    job: SpecMACS
+  - name: Logan Nguyen
+    job: Scientist
+  - name: Julia Santos
+    job: Ground contact
```

Comment:
`- name: Gunther Munther`
`job: HAMP calibration`

Reply:
I thought that happens on the ground and is rather considered as part of the maintenance? 🤔

Reply:
That's true... I wasn't so sure, if we should include the maintenance as well...? But if this would only confuses people then leave it out. 🙈

Reply:
Just to add to the confusion, the job "Ground contact" means that this is a person on ground, which is always reachable and is the first point of contact if there's something important with respect to the flight. We've had that role during EUREC4A and we've included the role in the flight plans such that everyone is aware about whom to ask in doubt.

But yes, maybe the calibration task is too unrelated to the flight plan, so I'd probably keep it out as well,

Reply:
I see. Alrighty. So we leave it like it is.